node-partial-builds/README.md

```diff
@@ -0,0 +1,5 @@
+# hosting-partial-builds
```

Comment:
Since this is not a hosting service, can you replace `hosting` with `node`?

Reply:
Woops, thought I had switched that